app/views/request/_share_by_link.html.erb

```diff
@@ -1,6 +1,6 @@
 <% if can? :share, info_request %>
   <div class="sidebar__section share-private-url">
-    <h2>Share with private link</h2>
+    <h2>_('Share with private link')</h2>
```

Comment:
Will need to be:

```suggestion
    <h2><%= _('Share with private link') %></h2>
```

Reply:
oops, shouldn't code review while on a call 🤦